src/components/common/MemosTable/AddReply/Reply.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState } from 'react';
 import FormInput from '../../FormInput';
-import axiosWithAuth from '../../../../utils/axiosWithAuth';
+import useAxiosWithAuth0 from '../../../../hooks/useAxiosWithAuth0';
```

Comment:
Understandably changing the import to utilize auth0.  Everything looks fine here.